cores/esp8266/libb64/cencode.cpp

```diff
@@ -23,10 +22,15 @@ void base64_init_encodestate_nonewlines(base64_encodestate* state_in){
   state_in->stepsnewline = -1;
 }
 
-char base64_encode_value(char value_in){
-  static const char encoding[] PROGMEM = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
-  if (value_in > 63) return '=';
-  return pgm_read_byte( &encoding[(int)value_in] );
+static inline char base64_encode_value(const char n) {
+  char r;
+
+  if (n < 26) r = n + 'A';
+  else if (n < 26 + 26) r = n - 26 + 'a';
+  else if (n < 26 + 26 + 10 ) r = n - 26 - 26 + '0';
+  else if (n == 62 ) r = '+';
+  else r = '/';
```

Comment:
No single-line if-elses. Please put the bodies on their own lines.

Reply:
changed, is this how you prefer it?

Reply:
No, I actually meant:
```cpp
if(blah1)
  bleh1;
else if (blah2)
  bleh2;
...
```

Braces are ok. In general, the preferred style for core code is Allman, while the Arduino-facing code (e.g.: examples) is Arduino style, which is a K&R variant.

Reply:
done Allman without braces ;-)